content/post/2020-07-24-data-science-tutorials-with-learnr-and-gradethis.md

```diff
@@ -0,0 +1,68 @@
+---
+title: Data Science Tutorials with Learnr and Gradethis
+author: Lee Suddaby, Zeno Kujawa
+date: '2020-07-24'
+slug: data-science-tutorials-with-learnr-and-gradethis
+categories: 
+  - teaching
+  - shiny
+  - rstats
+tags: []
+keywords:
+  - tech
+---
+
+*This post was contributed by [Lee Suddaby](https://github.com/lee-suddaby) and [Zeno Kujawa](https://github.com/ZenoMK).*
```

Comment:
Feel free to add one more sentence about each of you here if you like, like you're students at UoE, year, major, etc.